test/katex-spec.js

```diff
@@ -1037,17 +1040,42 @@ describe("A non-braced kern parser", function() {
     const emKern = "\\kern1em";
     const exKern = "\\kern 1 ex";
     const muKern = "\\kern 1mu";
+    const abKern1 = "a\\mkern1mub";
+    const abKern2 = "a\\kern-1mub";
+    const abKern3 = "a\\kern-1mu b";
```

Comment:
Thanks for filling out the tests.  Only `abKern3` would fail without the changes to Parser.js.